style.css

```diff
@@ -0,0 +1,544 @@
+/*! normalize.css v7.0.0 | MIT License | github.com/necolas/normalize.css */
+
+/* Document
+   ========================================================================== */
+
+/**
+ * 1. Correct the line height in all browsers.
+ * 2. Prevent adjustments of font size after orientation changes in
+ *    IE on Windows Phone and in iOS.
+ */
+
+html {
+  line-height: 1.15; /* 1 */
+  -ms-text-size-adjust: 100%; /* 2 */
+  -webkit-text-size-adjust: 100%; /* 2 */
+  font-size: 14px;
+  font-family: monospace;
+}
+
+/* Sections
+   ========================================================================== */
+
+div#bigbox {
+  width: 100%;
+  height: auto;
+  margin: auto;
+}
+
+div#nextbox {
+  width: 50%;
+  height: auto;
+  margin: auto;
+}
+
+.header {
+  font-size: 46px;
+  background-color: magenta;
```

Comment:
It's better to use HEX color instad the name of them. Browser could interpretate in differents ways